docs/sphinx/formats/dataset-table.txt

```diff
@@ -86,9 +92,15 @@ to open/import a dataset in a particular format.
    * - Canon RAW
      - .cr2, .crw, .jpg, .thm, .wav
      - Single file
+   * - CellH5 (HDF)
+     - .ch5
+     - Single file
    * - CellSens VSI
      - .vsi, .ets
      - One .vsi file and an optional directory with a similar name that contains at least one subdirectory with .ets files
+   * - CellVoyager
+     - .tif, .xml
+     - Directory with 2 master files 'MeasurementResult.xml' and 'MeasurementResult.ome.xml', used to stich together several TIF files.
```

Comment:
Noted. This will require a change to the parent class `CellVoyager.java`. Will push a commit shortly